### DISCUSSION TITLE
Is there a way to restart loxilb inside running docker ?

### DISCUSSION THREAD

**Discussion by TrekkieCoder** [loxilb-io/loxilb#59] [General]
Hello All,

I got loxilb docker running. Made some changes and want to rerun loxilb  without restarting loxilb docker itself. Is there a way to do this ?

Kindly help !!

**Comment by ghost** [comment]
Usually, the easy thing is to copy image to docker, commit it and then restart it. But if there is still a  need to restart loxilb only, I know of the following trick 

```
$ docker exec -it loxilb bash
# ip link del llb0; /root/loxilb-io/loxilb/loxilb --port=11112 --tls-port=8092
```
This basically spawns another loxilb. The original still keeps running without doing anything. We cant kill the original process since loxilb is the init process for the docker
